lib/CL/devices/ventus/pocl_ventus.cc

```diff
@@ -1093,8 +1093,10 @@ void pocl_ventus_write(void *data,
                        size_t size) {
   struct vt_device_data_t *d = (struct vt_device_data_t *)data;
   void *tmp_data = malloc(size);
```

Comment:
Is it more appropriate to put this line below the judgment statement of CL_MEM_USE_HOST_PTR?